notebooks/1_BayesianWorkflow/1_Ins_BayesRefresher.ipynb

```diff
@@ -0,0 +1,115 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Section 1: Bayesian refresher and Introduction to ArviZ"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "_What does an end to end Bayesian Workflow look like?_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Learning Objectives\n",
+    "* Refresh our understanding of Bayes Theorom\n",
+    "* Fit a small binomial model\n",
+    "* Show how a full statistical workflow, even outside of Bayesian Methods, requires more steps more than just model fitting\n",
+    "* Introduce ArviZ"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bayes Theorem\n",
+    "### The most common formulation\n",
+    "$$\n",
+    "\\Large\n",
+    "P(\\theta | y) = \\frac{ P(y) | \\theta)p(\\theta)}{p(y}\n",
+    "$$\n",
+    "\n",
+    "### Breaking it down\n",
+    "####  $P(\\theta | y)$ -> Posterior distribution\n",
+    "_\"What is the distribution of parameters given the observed data?\"_  \n",
+    "\n",
+    "After obtaining data, or making observations, what is our belief regarding the parameters of the underlying statistical model? \n",
+    "\n",
+    "* Estimating the posterior distribution is the goal of Bayesian analysis. \n",
+    "* The process of estimating the posterior distribution often refered to as **Inference**\n",
+    "* There are numerous inference ways to perform inference, each with their own pros and cons\n",
+    "    * In this tutorial we will only be using Markov Chain Monte Carlo (MCMC)\n",
+    "\n",
+    "####  $P(y | \\theta )$ -> Likelihood\n",
+    "_\"What is the probability of the observed data given a model parameter value\"_  \n",
+    "\n",
+    "Likelihood functions tell us how \"likely\" the observed data is, for all the possible parameter values. It is important to note that likelihood is not a probability distribution, it is a function that gives us probability values. Likelihood can roughly be thought of as loss functions from \"Machine Learning\"\n",
```

Comment:
"It is important to note that likelihood is not a probability distribution". Now sure about how useful is to clarify this, while the distinction is correct in practice we generally do not care, specially because we always use a likelihood together with a prior and we always get a proper posterior.